tests/test_split.py

```diff
@@ -0,0 +1,210 @@
+import petsc4py.PETSc
+
+petsc4py.PETSc.Sys.popErrorHandler() # noqa
```

Comment:
This is useful for debugging, I guess, but do we want to leave it in the unit tests?

Reply:
Right, let's cut that.